db/migrations/20181207111812-createAutomationTable.js

```diff
@@ -0,0 +1,43 @@
+module.exports = {
+  up: (queryInterface, DataTypes) => queryInterface.createTable('automation', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: DataTypes.INTEGER,
+    },
+    fellowName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    fellowId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    partnerName: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    partnerId: {
+      type: DataTypes.STRING,
+      allowNull: false,
+    },
+    type: {
+      type: DataTypes.ENUM,
+      values: ['onboarding', 'offboarding'],
```

Comment:
In the future, we might need to include other types. Is it possible to update this column on db through migration? If not, It might be best to use string type. Than enum  type can be configured at the model level, for flexibility